src/pages/[guild]/collect/[chain]/[address].tsx

```diff
@@ -281,6 +279,7 @@ const getStaticProps = async ({ params }) => {
 
   // Calling the serverless endpoint, so if we fetch this data for the first time, it'll be added to the Vercel cache
 
+  // WARNING: this environment variable probably doesn't exist
```

Comment:
Actually it's a [Vercel system environment variable](https://vercel.com/docs/projects/environment-variables/system-environment-variables), so it won't exist on localhost, only on Vercel. I think it's save to use it this way here (without including it in our custom env var validator) 👌